families/families.js

```diff
@@ -18,20 +19,12 @@ window.addEventListener('load', async () => {
 async function displayFamilies() {
     // fetch families from supabase
     const families = await getFamilies();
-    //console.log(families);
+    console.log(families);
     // clear out the familiesEl
     familiesEl.textContent = '';
 
     for (let family of families) {
```

Comment:
Here we would want to create an element and place the textContent = family.(name) to display each family. Then append this to the familyEl.